.github/workflows/update-pgxn-version.yml

```diff
@@ -16,21 +15,27 @@ jobs:
     name: Update pgxn configuration
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - name: Check out the repo
```

Comment:
standardized this step with other pipelines

Reply:
Is this the correct spelling though?

Reply:
You're right fixed it